components/code-editor/test/index.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { shallow } from 'enzyme';
+import { set, noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import CodeEditor from '../';
+
+describe( 'CodeEditor', () => {
+	it( 'should render without an error', () => {
+		set( global, 'wp.codeEditor.initialize', () => ( {
+			codemirror: {
+				on: noop,
+			},
+		} ) );
+
+		const wrapper = shallow( <CodeEditor value={ '<b>wowee</b>' } /> );
+		expect( wrapper.find( 'textarea' ) ).toHaveLength( 1 );
```

Comment:
We could use snapshot testing here, too. See example: https://github.com/WordPress/gutenberg/blob/300ca0a8acc73ff4013c0495a8c636527fcf588f/components/notice/test/index.js#L15 and docs: https://github.com/WordPress/gutenberg/blob/master/docs/testing-overview.md#snapshot-testing.

Reply:
Good idea. I've changed this to a snapshot test in 17dbeb231a901007b08e5fd0dd75840bdbbc1489.